index.js

```diff
@@ -1,27 +1,27 @@
-const menu = document.querySelector('.burger-menu')
-const closeItem = document.querySelector('.close-item')
-const aboutMenu = document.querySelector('.about-menu')
-const programMenu = document.querySelector('.porgram-menu')
-const speakersMenu = document.querySelector('.speak-menu')
-const partnersMenu = document.querySelector('.partners-menu')
-const contactMenu = document.querySelector('.contact-menu')
-const conferenceMenu = document.querySelector('.conference-menu')
+const menu = document.querySelector('.burger-menu');
+const closeItem = document.querySelector('.close-item');
+const aboutMenu = document.querySelector('.about-menu');
+const programMenu = document.querySelector('.porgram-menu');
+const speakersMenu = document.querySelector('.speak-menu');
+const partnersMenu = document.querySelector('.partners-menu');
+const contactMenu = document.querySelector('.contact-menu');
+const conferenceMenu = document.querySelector('.conference-menu');
```

Comment:
- [ ] On mobile screen, when on the about page, the list of your nav link doesn't contain the link to the homepage, instead, it contains the link to about page. Kindly replace the **`About`** with **`Home`**

Reply:
About page has been replaced with the home page.